src/ivory-px4-hw/test-client/calibration/calibrate.py

```diff
@@ -0,0 +1,173 @@
+#! /usr/bin/env python
+
+#  Copyright (C) 2010 Antoine Drouin
+#
+# This file is part of Paparazzi.
+#
+# Paparazzi is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
```

Comment:
Why not? If that is a problem, I can just add a link to the original file so people can download it.